docs/concepts/what-is-a-temporal-cron-job.md

```diff
@@ -90,9 +90,11 @@ Consider that using time zones in production introduces a surprising amount of c
 If you need to use time zones, here are a few edge cases to keep in mind:
 
 - **Beware Daylight Savings**: If a Temporal Cron Job is scheduled around the time when daylight saving time begins or ends, e.g. `30 2 * * *`, **it may run zero, one, or two times in a day**! The Cron library that we use does not do any special handling of DST transitions. Avoid schedules that include times that fall within DST transition periods.
```

Comment:
```suggestion
- **Beware Daylight Saving Time**: If a Temporal Cron Job is scheduled around the time when daylight saving time (DST) begins or ends (for example, `30 2 * * *`), **it might run zero, one, or two times in a day**! The Cron library that we use does not do any special handling of DST transitions. Avoid schedules that include times that fall within DST transition periods.
```